src/components/AccountList.tsx

```diff
@@ -54,20 +56,21 @@ const AccountList: React.FC = () => {
           const typeName = account.__typename
           return (
             <Item
-              key={id}
               title={title}
               active={isActive}
-              onClick={() => navigate(`/${projectPath}?type=account&id=${id}`)}
+              key={account.address}
             >
-              <AccountCard>
+              <AccountCard
+                onClick={() => navigate(`/${projectPath}?type=account&id=${id}&storage=${selectedResourceAccount || 'none'}`)}
+              >
                 <Avatar seed={project.seed} index={i} />
                 <Stack>
                   <strong>{accountAddress}</strong>
                   <small>{contractName || '--'}</small>
                 </Stack>
-
                 {isActive && <ExportButton id={account.id} typeName={typeName}/>}
               </AccountCard>
+              <ResourcesExplorerButton addr={accountAddress} />
```

Comment:
Can we use full `address` for a prop name? :)

Reply:
Changed in [this commit](https://github.com/onflow/flow-playground/pull/175/commits/289b5d04ba186213347bb27b87021fdfc8efaf3e)